src/Command/App/CreateCdeCommand.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Acquia\Cli\Command\App;
+
+use Acquia\Cli\Command\CommandBase;
+use Acquia\Cli\Exception\AcquiaCliException;
+use Acquia\Cli\Helpers\LoopHelper;
+use Acquia\Cli\Output\Checklist;
+use AcquiaCloudApi\Connector\Client;
+use AcquiaCloudApi\Endpoints\Databases;
+use AcquiaCloudApi\Endpoints\Environments;
+use AcquiaCloudApi\Endpoints\Ides;
+use AcquiaCloudApi\Endpoints\Notifications;
+use AcquiaCloudApi\Response\EnvironmentResponse;
+use AcquiaCloudApi\Response\IdeResponse;
+use AcquiaCloudApi\Response\OperationResponse;
+use React\EventLoop\Loop;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CreateCdeCommand.
+ */
+class CreateCdeCommand extends CommandBase {
+
+  protected static $defaultName = 'app:environment:create';
+
+  /**
+   * @var \Acquia\Cli\Output\Checklist
+   */
+  private Checklist $checklist;
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('Create a new Continuous Delivery Environment (CDE)');
+    $this->addArgument('label', InputArgument::REQUIRED, 'The label of the new environment');
+    $this->addArgument('branch', InputArgument::OPTIONAL, 'The vcs path (git branch name) to deploy to the new environment');
+    $this->acceptApplicationUuid();
```

Comment:
Infection is complaining that your automated tests don't include passing an application UUID as argument